src/mixins/formWatchers.js

```diff
@@ -21,10 +22,15 @@ export default {
     /**
      * Watch data changes
      * @param {object} data
+     * @param {array} changes
      */
-    watchDataChanges(data) {
+    watchDataChanges(data, changes) {
       if (this.watchers && this.watchers instanceof Array) {
```

Comment:
I think using `Array.isArray(this.watchers)` is more readable than an `instanceof` check. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray. So it should be possible to re-write this `if` block as:

```js
if (Array.isArray(this.watchers)) { ...
```

The initial check for `this.watchers` is unnecessary since a falsey value is not an array and would be caught by the `isArray` check.